addon/models/node.js

```diff
@@ -167,11 +167,11 @@ export default OsfModel.extend(FileItemMixin, {
         return promise;
     },
 
-    addChild(title, description, category, isPublic) {
+    addChild(title, description=null, category='project', isPublic=false) {
```

Comment:
Hmm, just realized that having `isPublic=false` as a default will probably not do the right thing for preprints.

Reply:
Yeah, changing interfaces is worrisome. Might be better to replace `category || 'project'` below with something like `category || category === '' ? category : 'project'`. Though that might be getting gross enough it's worth upgrading to 
```
if (!category && category !== '') {
    category = 'project';
}

Reply:
(or I guess `typeof(category) === 'undefined' ? 'project' : category`)